flash/core/classification.py

```diff
@@ -195,6 +195,7 @@ class FiftyOneLabels(ClassificationSerializer):
             list of FiftyOne labels (False)
     """
 
+    @requires("fiftyone")
```

Comment:
Neat !

Reply:
quick question. How does this behave with `inspect.signature` ?

Reply:
It should be fine as when the packages are installed the decorator is a no-op (just returns the original function). When not installed, the function will always error anyway but we use functools.wraps which preserves signatures :smiley: